containers/doc/rds.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Where is this file used? I didn't find any mention of it in this PR.

Reply:
Expected: this is just to store this in case we need to revive that thing. RDS DB setup will have to be done by the user like any other third party DB.